Example/lottie-ios/LottieRootViewController.m

```diff
@@ -74,7 +61,7 @@ - (void)_playLottieAnimation {
   [self.lottieLogo play];
 }
 
-#pragma mark -- UITableViewDataSource and Delegate
+#pragma mark - UITableViewDataSource and Delegate
```

Comment:
btw, i fixed these since the `--` shows up as "-" in the Xcode code navigator (pulldown), not a line.